models/engine/db_storage.py

```diff
@@ -74,3 +74,24 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """retrieve one object"""
+        statement = None
+        for clss in classes:
+            if cls is classes[clss] or cls is clss:
+                statement = select(classes[clss]).filter_by(id=id)
+                break
+
+        if statement is None:
+            return None
+        return self.__session.scalar(statement)
+
+    def count(self, cls=None):
+        """count objects"""
+        sum = 0
+        for clss in classes:
+            if cls is None or cls is classes[clss] or cls is clss:
+                objs = self.__session.query(classes[clss]).all()
+                sum += len(objs)
+        return (sum)
```

Comment:
It would be helpful to ensure that the function can handle cases where the class or ID provided isn't correct. Instead of causing an error or returning unexpected results, it could handle such situations by returning None.